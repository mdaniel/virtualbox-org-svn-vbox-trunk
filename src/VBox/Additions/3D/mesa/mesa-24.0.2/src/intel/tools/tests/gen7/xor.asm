xor(8)          g14<1>.xUD      g5.4<0>.zUD     g13<4>.xUD      { align16 1Q };
xor(8)          g47<1>UD        g45<8,8,1>UD    g46<8,8,1>UD    { align1 1Q };
xor(16)         g87<1>UD        g83<8,8,1>UD    g85<8,8,1>UD    { align1 1H };
xor(8)          g124<1>UD       g5<8,8,1>UD     0x000003ffUD    { align1 1Q };
xor(16)         g120<1>UD       g13<8,8,1>UD    0x000003ffUD    { align1 1H };
