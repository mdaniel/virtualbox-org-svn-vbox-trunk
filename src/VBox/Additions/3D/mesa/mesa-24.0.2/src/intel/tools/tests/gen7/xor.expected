07 01 60 00 21 04 c1 21 ba 00 0a 00 a0 01 60 00
07 00 60 00 21 04 e0 25 a0 05 8d 00 c0 05 8d 00
07 00 80 00 21 04 e0 2a 60 0a 8d 00 a0 0a 8d 00
07 00 60 00 21 0c 80 2f a0 00 8d 00 ff 03 00 00
07 00 80 00 21 0c 00 2f a0 01 8d 00 ff 03 00 00
